src/oemof/solph/_energy_system.py

```diff
@@ -176,12 +176,12 @@ def __init__(
         self._extract_periods_matrix()
 
     def _extract_periods_years(self):
-        """Map simulation years to the respective period based on time indices
+        """Map years in optimization to respective period based on time indices
 
         Returns
         -------
         periods_years: dict
-            the simulation year of the start of each a period,
+            year of the start of each period,
             relative to the start of the optimization run and starting with 0
```

Comment:
Two things:
- function does not return anything, but sets member variable instead - so `Returns` section is not needed, right?
- if you want to use `Returns` section anyway or want to mention `period_years`, `periods_years` should be of type `list`

Reply:
Thank you for pointing this out. I resolved it. Please have a look.